src/NuGet.Core/NuGet.Commands/RestoreCommand/Diagnostics/UnresolvedMessages.cs

```diff
@@ -107,6 +121,15 @@ internal static async Task LogAsync(IList<DownloadDependencyResolutionResult> do
                                                   .OrderBy(e => e, StringComparer.OrdinalIgnoreCase));
 
                     message = string.Format(CultureInfo.CurrentCulture, Strings.Error_NoPackageVersionsExist, unresolved.Name, sourceList);
+
+                    if (isPackageSourceMappingEnabled && applicableRemoteLibraryProviders.Count != allRemoteLibraryProviders.Count)
+                    {
+                        var notConsideredSourceList = FormatProviderNames(GetUnusedLibraryProviders(applicableRemoteLibraryProviders, allRemoteLibraryProviders));
```

Comment:
Instead of what not considered, should be display what is considered?
I saw some repos have 10-20 sources defined then customer seeing this warning may see all those sources except 1 here, could spam customer with bunch of details.
I don't have strong opinion here, feels 2 sides of same coin in most cases.

Reply:
The considered sources are already included. 

This change in particular is for NU1101. Details in the description. 

In particular this warning with source mapping enabled now would be:

> error NU1101: Unable to find package NuGet.VersioningBla. No packages exist with this id in source(s): nuget-build. PackageSourceMapping is enabled, the following source(s)
 were not considered: nuget.

Reply:
In general though, the changes in this PR all include the addition of sources not considered as the considered ones are already part of the messages. 

In other cases, like NU1100, no sources were considered.